tl/controllers/request-response.rst

```diff
@@ -292,278 +292,278 @@ Mayroong ilang mga built-in na mga detektor na magagamit mo:
+isang partikular na uri ng nilalaman.
 
-Get all types::
+Kunin ang lahat ng mga uri::
 
     $accepts = $this->request->accepts();
 
-Check for a single type::
+Suriin para sa isang solong uri::
 
     $acceptsJson = $this->request->accepts('application/json');
 
 .. php:method:: acceptLanguage($language = null)
 
-Get all the languages accepted by the client,
-or check whether a specific language is accepted.
+Kunin ang lahat ng mga language na tinatanggap sa kliyente,
+o suriin kung ang tiyak na language ay tinatanggap.
 
-Get the list of accepted languages::
+Kunin ang listahan sa tinatanggap na mga language::
 
     $acceptsLanguages = $this->request->acceptLanguage();
 
-Check whether a specific language is accepted::
+Suriin kung ang isang tiyak na language ay tinatanggap::
 
     $acceptsSpanish = $this->request->acceptLanguage('es-es');
 
 .. _request-cookies:
 
-Cookies
--------
+Mga Cookie
+----------
 
-Request cookies can be read through a number of methods::
+Hiling na mga cookie ay maaaring mabasa gamit ang isang bilang ng mga pamamaraan::
 
-    // Get the cookie value, or null if the cookie is missing.
+    // Kunin ang cookie na halaga, o null kung ang cookie ay nawawala.
     $rememberMe = $this->request->getCookie('remember_me');
 
-    // Read the value, or get the default of 0
+    // Basahin ang halaga, o kunin ang default sa 0
     $rememberMe = $this->request->getCookie('remember_me', 0);
 
-    // Get all cookies as an hash
+    // Kunin ang lahat ng mga cookie bilang isang hash
     $cookies = $this->request->getCookieParams();
 
-    // Get a CookieCollection instance (starting with 3.5.0)
+    // Kunin ang CookieCollection na instance (na nagsisimula sa 3.5.0)
     $cookies = $this->request->getCookieCollection()
 
-See the :php:class:`Cake\\Http\\Cookie\\CookieCollection` documentation for how
-to work with cookie collection.
+Tingnan ang :php:class:`Cake\\Http\\Cookie\\CookieCollection` na dokumentasyon para sa kung paano
+ipagana gamit ang koleksyon ng cookie.
 
 .. versionadded:: 3.5.0
     ``ServerRequest::getCookieCollection()`` was added in 3.5.0
 
 .. index:: $this->response
 
-Response
-========
+Tugon
+=====
 
 .. php:class:: Response
 
-:php:class:`Cake\\Http\\Response` is the default response class in CakePHP.
-It encapsulates a number of features and functionality for generating HTTP
-responses in your application. It also assists in testing, as it can be
-mocked/stubbed allowing you to inspect headers that will be sent.
-Like :php:class:`Cake\\Http\\ServerRequest`, :php:class:`Cake\\Http\\Response`
-consolidates a number of methods previously found on :php:class:`Controller`,
-:php:class:`RequestHandlerComponent` and :php:class:`Dispatcher`. The old
-methods are deprecated in favour of using :php:class:`Cake\\Http\\Response`.
+Ang :php:class:`Cake\\Http\\Response` ay isang default na tugon na class sa CakePHP.
+Ito ay nagpapaikut-ikot sa bilang ng mga tampok atfunctionality para bumuo ng HTTP
```

Comment:
- please change `nagpapaikut-ikot` to `nag-eencapsulate` instead.
- `sa ilang mga tampok at functionality para sa pagbubuo ng HTTP`

Reply:
done change